controllers/dscinitialization/dscinitialization_test.go

```diff
@@ -227,7 +227,7 @@ func namespaceExists(ns string, obj client.Object) func() bool {
 	}
 }
 
-func objectExists(ns string, name string, obj client.Object) func() bool { //nolint
+func objectExists(ns string, name string, obj client.Object) func() bool { //nolint:unparam
```

Comment:
Why is it complaining about unused parameters?

Reply:
It complains about ``objectExists` - `name` always receives `applicationNamespace` (`"test-application-ns"`)`
But we may reuse this functions for a different name in the future right?

Reply:
Or perhaps we re-work tests so that they use their own namespaces instead of a global one? :)